Rakefile

```diff
@@ -352,11 +347,6 @@ end
 
 # Misc Tasks ---------------------------------------------------------
 
-desc "Cleanup trailing whitespace"
-task :whitespace do
-  system 'find . -not \( -name .svn -prune -o -name .git -prune \) -type f -print0 | xargs -0 sed -i "" -E "s/[[:space:]]*$//"'
-end
-
```

Comment:
Why is this task duplicated? You mean by rubocop? Technically this task works on all project files, so if we add some traling whitespace to the README, this task will remove it.

Reply:
Fmm, Your comment is right. But I never invoke this task in the past releases.

Reply:
Right. We should either remove it or use it, yeah. Maybe add it to CI? I don't have a strong opinion, whatever you think it's best works for me :+1:.